lib/krikri/mapper.rb

```diff
@@ -133,8 +133,13 @@ def run(activity_uri = nil)
             rec.mint_id! if rec.node?
             activity_uri ? rec.save_with_provenance(activity_uri) : rec.save
           rescue => e
-            Rails.logger.error("Error saving record: #{rec.rdf_subject}\n" \
-                               "#{e.message}\n#{e.backtrace}")
+            if rec
+              Rails.logger.error("Error saving record: #{rec.rdf_subject}\n"\
```

Comment:
Because this is a `SoftwareAgent`, you should be able to change this to `log :error, "message..."` and get the benefit of tagged logging with a timestamp.